src/ducks/Watchlists/Actions/Edit/EditForm/index.js

```diff
@@ -48,26 +51,42 @@ const EditForm = ({
       name === defaultSettings.name &&
       description === defaultSettings.description &&
       isPublic === defaultSettings.isPublic &&
+      listItems === defaultSettings.listItems &&
       id
     ) {
       toggleOpen(false)
     } else {
-      onFormSubmit({ name, description, isPublic })
+      onFormSubmit({ name, description, isPublic, listItems })
     }
   }
 
+  function checkIsTouched (key, value) {
+    const _value =
+      value === '' || value === undefined || value === null ? null : value
+    const _formState = { ...formState, [key]: _value }
+    _formState.description =
+      _formState.description === '' ? null : _formState.description
```

Comment:
Better to name it `newFormState` since `_` prefix is mostly used for ignoring variable in tools like eslint